src/styles/blocks/calendar.scss

```
@@ -0,0 +1,74 @@
+.calendar {
+  height: 100vh;
+
+  &__title {
+    display: flex;
+    justify-content: center;
+
+    padding-top: 25px;
+
+    font-size: 55px;
+    color: black;
+    cursor: pointer;
+
+    &:hover {
+      text-shadow: 3px 3px pink;
+    }
+  }
+
+  &__content {
+    display: flex;
+    flex-wrap: wrap;
+    gap: $day-gap;
+
+    max-width: $calendar-width;
+
+    padding: 25px $calendar-padding;
+
+    margin: auto;
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-grow: 0;
+    flex-shrink: 0;
+    flex-basis: $day-square-size;
+
+    height: $day-square-size;
+
+    text-align: center;
+    border: 1px solid black;
+
+    background-color: $day-color;
+
+    cursor: pointer;
+    transition: background-color 0.5s, transform 0.5s;
```

```suggestion
    transition: 0.5s;
```

Try to set only duration - it'll be applied to any changes in attributes (same as special word `all`). Yuor shadow will work better on hover.

---

Is true, thanks